config/environments/development.rb

```diff
@@ -34,14 +36,22 @@
   config.action_mailer.raise_delivery_errors = false
 
   config.action_mailer.perform_caching = false
-  config.action_mailer.default_url_options = {
-    host: 'localhost',
-    port: '3000'
-  }
 
   # Print deprecation notices to the Rails logger.
   config.active_support.deprecation = :log
 
+  # Raise exceptions for disallowed deprecations.
+  config.active_support.disallowed_deprecation = :raise
+
+  # Tell Active Support which deprecation messages to disallow.
+  config.active_support.disallowed_deprecation_warnings = []
+
+  # Raise an error on page load if there are pending migrations.
+  config.active_record.migration_error = :page_load
+
+  # Highlight code that triggered database queries in logs.
+  config.active_record.verbose_query_logs = true
```

Comment:
Might be rails 6.1.'s best feature.

```
[DEBUG]   User Load (0.5ms)  SELECT "users".* FROM "users"
[DEBUG]   ↳ app/views/users/_users_table.html.erb:14
```

Reply:
oooo, I also like the deprecation strictness!